index.html

```diff
@@ -6,59 +6,244 @@
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project4.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                    Tonic
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project2.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                    Multi-Post Stories Gain+Glory
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+        </div>
+    </section>
+    <section class="container-flex about-myself">
+        <div class="container">
+            <h1>About me</h1>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <button type="button">
+                Get My Resume
+            </button>
+        </div>
+        <ul class="categorie-about-myself">
             <li>
-                <a href="">
-                    <img src="./assets/social-media/medium.svg" alt="medium icon">
-                </a>
+                <article class="card-categories">
+                    <div class="categorie-image">
+                        <img src="./assets/icon-languages.svg" alt="icon languages">
+                    </div>
+                    <h2 class="categorie-title">
+                        Languages
+                    </h2>
+                    <ul class="categorie-tags">
+                        <li>React</li>
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
```

Comment:
- [ ] Please fix the margin here and please make sure that you are following the desing correctly 
- this screenshot was token for 320px screens
![image](https://user-images.githubusercontent.com/74744787/159080982-004b82ca-2a4c-41df-8f84-f835e743d0bf.png)